src/simsopt/configs/zoo.py

```diff
@@ -198,38 +200,64 @@ def get_w7x_data(Nt_coils=48, Nt_ma=10, ppp=2):
     """
-    Download a configuration from the QUASR database.
+    Download a configuration from the QUASR database.  Downloaded configuration files are cached in 
+    [SIMSOPT_INSTALL_DIR]/src/simsopt/configs/QUASR_cache/
 
     Args:
-        ID: the ID of the configuration to download.  The database is navigatable at https://quasr.flatironinstitute.org/
-            Alternatively, you can download the latest full set of devices from https://zenodo.org/doi/10.5281/zenodo.10050655
+        ID (int): the ID of the configuration to download.  A pandas dataframe containing metadata on the devices, including all
+                  valid ID numbers is located at: https://quasr.flatironinstitute.org/QUASR.pkl
+                  The database is navigatable online at https://quasr.flatironinstitute.org/
+                  Alternatively, you can download the latest full set of devices from https://zenodo.org/doi/10.5281/zenodo.10050655
+
+        return_style (string, optional): 'simsopt-style' or 'quasr-style', defaults to 'quasr-style'.
+                      For 'simsopt-style', we return [curves_halfperiod, currents] where the curves lie on a half period, similar to get_ncsx_data().
+                      You can take the currents and apply symmetries to get the full device. NOTE: this does not return the magnetic axis. 
+                      quasr-style: [coils_full_torus, surfaces], returns all coils and all surfaces in the object. 
         
-        return_style: 'simsopt-style' or 'quasr-style'. '. 
-                      simsopt-style: [coils_1fp, surface], similar to get_ncsx_data(), gives the curves and currenst for one field period,
-                      which you can copy and rotate using simsopt methods (allows finer control over degrees-of-freedom). 
-                      NOTE: this does not return the magnetic axis. 
-                      quasr-style: [coils_all, surfaces], returns all coils and all surfaces in the object. 
+        verbose (boolean): if true, additional caching and downloading status messages are printed, otherwise they are not.
 
         returns: depending on return_style: 
            simsopt-style: [list of simsopt.geo.Coil objects, list of simsopt.field.Current objects]
-           quasr-style: [list of simsopt.geo.SurfaceXYZTensorFourier objects, list of simsopt.field.COIL objects]
+           quasr-style: [list of simsopt.geo.SurfaceXYZTensorFourier objects, list of simsopt.field.Coil objects]
     """
     
     if return_style not in ['simsopt-style', 'quasr-style']:
         raise ValueError(f"invalid return_style: {return_style}, must be either simsopt-style or quasr-style")
     
-    id_str = f"{ID:07d}"
-    # string to 7 digits
+    id_str = f"{ID:07d}" # string to 7 digits
     url = f'https://quasr.flatironinstitute.org/simsopt_serials/{id_str[0:4]}/serial{id_str}.json'
+    
+    FILE_PATH = THIS_DIR / f'QUASR_cache/serial{id_str}.json'
```

Comment:
I'm unsure if this will still do the right thing on environments where `/` is not the path separator (e.g. windows). If you care, could change to `FILE_PATH = THIS_DIR / 'QUASR_cache' / f"serial{id_str}.json"` (and push it onto `pathlib`).